src/main/java/emu/grasscutter/command/commands/ClearCommand.java

```diff
@@ -8,28 +8,21 @@
 import emu.grasscutter.game.player.Player;
 
 import java.util.List;
-import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Stream;
 
+import static emu.grasscutter.command.CommandHelpers.lvlRegex;
+import static emu.grasscutter.command.CommandHelpers.matchIntOrNeg;
+
 @Command(
     label = "clear",
     usage = {"(all|wp|art|mat) [lv<max level>] [r<max refinement>] [<max rarity>*]"},
     permission = "player.clearinv",
     permissionTargeted = "player.clearinv.others")
 public final class ClearCommand implements CommandHandler {
-    private static Pattern lvlRegex = Pattern.compile("l(?:vl?)?(\\d+)");  // Java doesn't have raw string literals :(
     private static Pattern refineRegex = Pattern.compile("r(\\d+)");
     private static Pattern rankRegex = Pattern.compile("(\\d+)\\*");
```

Comment:
```suggestion
```

May as well move the rest of the regexes into `CommandHelpers` too

Reply:
sound good, I will do that later